### PR TITLE
Median MOE Fix Update

### PR DESCRIPTION
**Pull request by sastoudt**
Based on census feedback:

- add sampling percentage as user input (makes finite sample correction, values differ by data source)
- includes tests 
- includes update to docs

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -94,14 +94,14 @@ The minimum value in the first range and the maximum value in the last range can
...
-
+For a margin of error to be returned, a "design factor" and sampling percentage must be provided to calculate the standard error. These statistical inputs are used to tailor the estimate to the variance of the dataset and correct for a finite sample. The Census Bureau publishes design factors as part of its PUMS Accuracy statement. Find the value for the dataset you are estimating by referring to `the bureau's reference material <https://www.census.gov/programs-surveys/acs/technical-documentation/pums/documentation.html>`_. 
+For the sampling percentage value, the 1-year ACS is designed to be a 2.5% sample of the population, and the 1-year PUMS is designed to be a 1% sample of the population. You can multiply these percentages by 5 for the 5-year versions.
 .. code-block:: python
 
-  >>> census_data_aggregator.approximate_median(income, design_factor=1.5)
+  >>> census_data_aggregator.approximate_median(income, design_factor=1.5, sampling_percentage=1)
```
Comment (palewire) [inline README.rst]: Add one example for each survey, acs1, acs5, pums

Comment (palewire) [reply README.rst]: Do we know ACS3?

File: census_data_aggregator/__init__.py [hunk 2]
```diff
@@ -79,6 +79,9 @@ def approximate_median(range_list, design_factor=None):
             variance of the dataset. The Census Bureau publishes design factors as part of its PUMS Accuracy statement.
             Find the value for the dataset you are estimating by referring to `the bureau's reference material`_.
             If you do not provide this input, a margin of error will not be returned.
+        sampling_percentage (float, optional): A statistical input used to correct variance for finite population.
```
Comment (palewire) [inline census_data_aggregator/__init__.py]: Can we use a more casual term than finite population? The percentage of the population that was in the survey sample.

**Comment by sastoudt** [comment]
I think this version is now technically correct. It is missing a PUMS example, but I think there needs to be a separate function for that. The current approximation would work, but you could do better by just using the individual data you have from PUMS instead of inputting bins. If we merge this PR, then I can start fresh and break things up into ```approximate_median_acs``` (wouldn't need the design factor anymore) and ```approximate_median_pums```.

**Comment by sastoudt** [comment]
Proof of concept of separating ```approximate_median_acs``` and ```approximate_median_pums``` [here](https://github.com/sastoudt/census-data-aggregator/blob/sepACS_PUMS/sepACS_PUMS.ipynb). For the more direct PUMS calculation I tried a weighted median and its MOE calculation, but I need to double check that this math is kosher.

**Pull request merged**